trustkit/src/main/java/com/datatheorem/android/trustkit/config/TrustKitConfigurationParser.java

```diff
@@ -160,8 +161,8 @@ private static PinSetTag readPinSet(@NonNull XmlPullParser parser) throws IOExce
     }
 
     private static class TrustkitConfigTag {
-        Boolean enforcePinning = null;
-        Boolean disableDefaultReportUri = null;
+        Boolean enforcePinning = false;
+        Boolean disableDefaultReportUri;
```

Comment:
Why this change? I think my intent here was to explicitely not initialize these config variables.